src/main/java/net/minecraftforge/client/IItemDecorator.java

```diff
@@ -19,11 +21,11 @@
 {
 
     /**
-     * Is called after {@linkplain ItemRenderer#renderGuiItemDecorations(Font, ItemStack, int, int, String)} is done rendering.
+     * Is called after {@linkplain ItemRenderer#renderGuiItemDecorations(PoseStack, Font, ItemStack, int, int, String)} is done rendering.
      * The StackCount is rendered at blitOffset+200 so use the blitOffset with caution.
      * <p>
      * The RenderState during this call will be: enableTexture, enableDepthTest, enableBlend and defaultBlendFunc
      * @return true if you have modified the RenderState and it has to be reset for other ItemDecorators
      */
-    boolean render(Font font, ItemStack stack, int xOffset, int yOffset, float blitOffset);
+    boolean render(PoseStack poseStack, Font font, ItemStack stack, int xOffset, int yOffset);
```

Comment:
Please make this a default method and implement it as a bouncer to the method with the signature. That should prevent breaking changes.

Reply:
Allright! Should i mark the old one as deprecated then?

Reply:
yes